bubblewrap.c

```diff
@@ -2216,6 +2237,13 @@ main (int    argc,
       prctl (PR_SET_SECCOMP, SECCOMP_MODE_FILTER, &seccomp_prog) != 0)
     die_with_error ("prctl(PR_SET_SECCOMP)");
 
+  /* Optionally limit memory. This must occur after the fork and
+   * before the exec* call. */
+  if (opt_limit_mem) {
```

Comment:
This needs to come *before* the seccomp filter - see the rationale in https://github.com/projectatomic/bubblewrap/issues/155